crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst

```diff
@@ -9,6 +9,43 @@ Changelog - nrf_cc3xx_mbedcrypto
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_mbedcrypto - 0.9.16
+*****************************
+
+New version of the runtime library with the following bug fixes and improvements:
+
+* Built with MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER enabled to ensure support with and without TF-M.
+* Added support for zero input message length for EDDSA for RFC test compliance.
+* Removed unused trace functions from runtime library.
+
+Library built against Mbed TLS version 3.1.0.
+
+This version is dependent on the nrf_cc310_platform or nrf_cc312_platform library for low-level initialization of the system and proper RTOS integration.
+
```

Comment:
```suggestion
```

This is not a change either.

Reply:
This is not a change statement

Reply:
Then it shouldn't be in the changelog.

Reply:
No. It is an essential note about the runtime-library and it hurts nobody to mention it

Reply:
If it is essential information then it should go in the documentation, not the changelog.

The negative consequences of having it in the changelog is that it is being communicated that this has changed, and also we are bloating the changelog with documentation that is constant for every release.

Reply:
Won't change in this PR, discussions to be had offline on the topic of documentation and changelog